src/docfx/template/XrefTagParser.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class XrefTagParser
```

Comment:
Consider prefix it with `Mustache` to indicate this is not a general xref tag parser

```suggestion
    internal static class MustacheXrefTagParser
```